functions/public/Invoke-WPFUpdateMGMGT.ps1

```diff
@@ -0,0 +1,58 @@
+function Invoke-WPFUpdateMGMT {
+    param (
+        [switch]$Selected,
+        [switch]$All
+    )
+
```

Comment:
```suggestion

    if ((-not $Selected -and -not $All) -or ($Selected -and $All)) {
        Write-Host "[Invoke-WPFUpdateMGMT] Use either 'Selected' or 'All' switches, used switches are: 'Selected' is $Selected, 'All' is $All" -ForegroundColor Red
        return
    }

```
Adding a check before proceeding helps eliminate possible misuse of this function.